findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
Hmm, `Messages.properties` keys matching `[A-Z].+` are quite common, so I wonder why only this plugin would be affected.

The generator could add

```java
@SuppressFBWarnings("NM_METHOD_NAMING_CONVENTION")
```

if it can detect that the FindBugs annotations library is in the classpath of the project (which it should be for any plugin using the 2.x parent).